{"name": "test_login_fail[测试用户名正确, 密码为空用户名:linux超密码期望值{'code': 1002, 'msg': '密码不能为空', 'data': None}]", "status": "passed", "description": "测试用户名或密码错误", "steps": [{"name": "输入用户名", "status": "passed", "parameters": [{"name": "user", "value": "'linux超'"}], "start": 1582701229400, "stop": 1582701229400}, {"name": "输入密码", "status": "passed", "parameters": [{"name": "pwd", "value": "''"}], "start": 1582701229401, "stop": 1582701229401}], "parameters": [{"name": "data", "value": "{'case': '用户名正确, 密码为空', 'user': 'linux超', 'pwd': '', 'expected': {'code': 1002, 'msg': '密码不能为空', 'data': None}}"}], "start": 1582701229400, "stop": 1582701229401, "uuid": "7ccd7184-3b16-48e1-af65-cc76af0c8df2", "historyId": "f0f08d9d3b58fbc1a6d68d10e6e7e034", "testCaseId": "23bc14a2becd8ad6fc8bdb5969b41867", "fullName": "test_allure_feature.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "测试登录失败"}, {"name": "suite", "value": "test_allure_feature"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-1TP29R9"}, {"name": "thread", "value": "14420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure_feature"}]}